nodes/src/nodes/vectordb_postgres/vectordb_postgres.py

```
@@ -51,23 +51,7 @@
 SQL_QUERIES = {
     'check_collection_exists': "SELECT EXISTS (SELECT FROM pg_tables WHERE schemaname = 'public' AND tablename = %s)",
     # fmt: off
```

Ruff (0.15.7)

[warning] This suppression comment is invalid because it cannot be in an expression, pattern, argument list, or other non-statement

Remove this comment

(RUF028)